src/events/messages.ts

```diff
@@ -182,9 +182,15 @@ tmi.on(
 
       if (message === Giveaway.commands.entrants) {
         if(Giveaway.isOpen) {
-          tmi.say(config.channel, Giveaway.getCurrentEntrantsMessage())
+          if(Giveaway.entrants.size <= 0)
```

Comment:
No idea whats going on here with the line indentation, I did it in a hurry lol

Reply:
Please allow mods to use this command as discussed :)